use-cases/website-authorization/docs/DEPLOYMENT.md

```diff
@@ -0,0 +1,140 @@
+# Overview
+
+This sample solution is a web application, currently available in Python with Flask. 
+In this guide, we demo how to deploy and host the solution on AWS Elastic Beanstalk platform. Alternatively, the application can be run locally and, steps and tips are provided at the end of this guide.
+
+# Solution
+
+This sample solution is an implementation of the Login with Amazon website authorization workflow. This web app can be deployed as a sample developer website and configured with your developer application to initiate the authorization workflow, from the selling partner clicking on an Authorize button until retrieving the Refresh Token and executing a first successful API call.
+
+We recommend the following resources for more details on authorization with the Selling Partner API:
+[Authorizing Selling Partner API Applications](https://developer-docs.amazon.com/sp-api/docs/authorizing-selling-partner-api-applications)
+[Website Authorization Workflow](https://developer-docs.amazon.com/sp-api/docs/website-authorization-workflow)
+
+# Pre-requisites
+
+The pre-requisites for deploying the Sample Solution App to AWS are:
+* [Registering as a developer for SP-API](https://developer-docs.amazon.com/sp-api/docs/registering-as-a-developer), and [registering an SP-API application](https://developer-docs.amazon.com/sp-api/docs/registering-your-application).
+* An AWS account
+* An [IAM user](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_users.html) with a permissions policy to create, deploy and manage an Elastic Beanstalk deployment
+  * An example policy is provided below
+* The [AWS CLI](https://aws.amazon.com/cli/)
+  * If not present, it will be installed as part of the deployment script.
+* The [EB CLI](https://docs.aws.amazon.com/elasticbeanstalk/latest/dg/eb-cli3.html)
+  * If not present, it will be installed as part of the deployment script.
+* An https enabled URL to the deployed application configured as OAuth redirect URL in your application profile on Developer Portal
+* Python 3 installed
+
+# Usage
+### 1. Update the application variables
+* In *code/python/utils/api_utils.py*, update the CLIENT_ID and CLIENT_SECRET with the values for your Developer Central application.
```

Comment:
I think this should be `code/python/utils/constants.py`. Same for APPLICATION_ID below.

Also, you can use a link to the file so it can be accessed with the click of a button.